swaggen/src/main/java/annotation/SwaggerGenClass.java

```diff
@@ -0,0 +1,71 @@
+package annotation;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.ElementType;
+
+/**
+ * SwaggerGen class level annotation description.
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface SwaggerGenClass {
+  /**
+   * The base path
+   * 
+   * @return the base path
+   */
+  String basePath() default "";
+  	/**
```

Comment:
Clearly we are at the point where we need to discuss tabs versus spaces. I'm not going to insist that this be fixed now, since once we have some formatting standards we will publish them and run an auto-formatter over the whole project.